local_pathfinding/objectives.py

```diff
@@ -20,61 +29,322 @@ def __init__(self, space_information):
         self.space_information = space_information
 
     def motionCost(self, s1: ob.SE2StateSpace, s2: ob.SE2StateSpace) -> ob.Cost:
-        raise NotImplementedError
+        return ob.Cost(0.0)
```

Comment:
Don't we want to raise a `NontImplementedError` if an objective doesn't implement `motionCost`?